.github/workflows/release-with-changesets.yml

```diff
@@ -19,14 +15,15 @@ on:
 
 jobs:
   test-nodejs:
-    # We just check the message of first commit as there is always just one commit because we squash into one before merging
-    # "commits" contains array of objects where one of the properties is commit "message"
+    # We just check the message of the first commit as there is always just one commit because we squash into one before merging
+    # "commits" contains an array of objects where one of the properties is the commit "message"
     # Release workflow will be skipped if release conventional commits are not used
     if: |
       startsWith( github.repository, 'asyncapi/' ) &&
       (startsWith( github.event.commits[0].message , 'fix:' ) ||
       startsWith( github.event.commits[0].message, 'fix!:' ) ||
       startsWith( github.event.commits[0].message, 'feat:' ) ||
+      startsWith( github.event.commits[0].message, 'chore(release):' ) ||
```

Comment:
I don't get why we need it here?

Reply:
when pr will be created by ci the commit message will be chore(release) if we dont have here the ci will not run. as the commit message is any of fix and feat.

Reply:
ok, I'll suggest a change in this workflow - a comment to yaml - that explains it, and you will have a look and commit if I'm right - then we know if we are on the same page